src/components/VanillaEvents.tsx

```diff
@@ -156,11 +157,24 @@ const VanillaEvents = () => {
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const [upcoming, past] = await Promise.all([
-          getUpcomingEvents(),
-          getPastEvents()
-        ]);
-
+        // Fetch all events
+        const events = await getAllEvents();
```

Comment:
when we can use the `new Date()` here, why dont we pass the date towards the function `getAllEvents` as parameter and really just query the needed timeframe?